descwl_coadd/tests/test_coadd.py

```diff
@@ -47,7 +47,7 @@ def _make_sim(
     )
 
     if se_dim is None:
-        se_dim = get_se_dim(coadd_dim=coadd_dim, dither=dither, rotate=rotate)
+        se_dim = get_se_dim(coadd_dim=coadd_dim, rotate=rotate)
```

Comment:
What is this change about?

Reply:
See https://github.com/LSSTDESC/descwl-shear-sims/pull/220

Reply:
Specifically https://github.com/LSSTDESC/descwl-shear-sims/pull/220#discussion_r1888931295